plugin/package.json

```diff
@@ -11,14 +11,14 @@
   "scripts": {
     "start:datacatalog": "vite -c vite.config.datacatalog-web.ts",
     "start:sidebar": "vite -c vite.config.sidebar-web.ts",
-    "build": "run-s typecheck clean build:web build:plugin",
+    "build": "run-s typecheck build:web build:plugin",
     "build:plugin": "run-s build:sidebar build:datacatalog",
     "build:datacatalog": "vite build -c vite.config.datacatalog.ts",
     "build:sidebar": "vite build -c vite.config.sidebar.ts",
-    "build:web": "run-s clean:build build:datacatalog-web build:sidebar-web --emptyOutDir false",
-    "build:datacatalog-web": "vite build -c vite.config.datacatalog-web.ts --emptyOutDir false",
-    "build:sidebar-web": "vite build -c vite.config.sidebar-web.ts --emptyOutDir false",
-    "clean": "del dist",
+    "build:web": "run-s  build:datacatalog-web build:sidebar-web",
+    "build:datacatalog-web": "vite build -c vite.config.datacatalog-web.ts",
+    "build:sidebar-web": "vite build -c vite.config.sidebar-web.ts",
+    "clean": "del dist/",
```

Comment:
Did you modify these for any specific reason? Or just because your old PR changed this? If there were no actual reasons, please keep this as it was.

Reply:
https://github.com/eukarya-inc/reearth-plateauview/pull/29/commits/27901afd519e051b965fe178e24e7ffaa7d9b1a1

Reply:
https://github.com/eukarya-inc/reearth-plateauview/tree/2bc9d5dcaf9b150f2d7b28bf6f01ec143945d760